pulsar-client-api/src/main/java/org/apache/pulsar/client/api/Consumer.java

```diff
@@ -523,11 +523,15 @@ CompletableFuture<Void> reconsumeLaterCumulativeAsync(Message<?> message,
 
     /**
      * The asynchronous version of {@link Consumer#seek(MessageId)}.
+     * <p>
+     * If there is already a seek operation in progress, the method will log a warning and return a canceled future.
```

Comment:
```suggestion
     * If there is already a seek operation in progress, the method will log a warning and return a future completed exceptionally.
```

See https://github.com/apache/pulsar/pull/20321